wandb/errors/__init__.py

```diff
@@ -1,8 +1,16 @@
 __all__ = [
     "Error",
-    "UsageError",
     "CommError",
+    "TimeoutError",
+    "PermissionsError",
+    "AuthenticationError",
+    "AuthorizationError",
+    "UsageError",
+    "InvalidError",
     "UnsupportedError",
+    "ConfigurationError",
+    "DependencyError",
```

Comment:
Note to selves: add an error path if the protobuf library version does not match our requirements.